### PR TITLE
Update SmartSwitch config to align YANG standard

### PR DESCRIPTION
**Pull request by wen587**
Fixes: https://github.com/sonic-net/sonic-buildimage/issues/20730
#### Why I did it
The generated t1 config fails YANG validation, which leads to config setup failure since we enforce YANG validation in config reload. 
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Update config to align with YANG 
#### How to verify it
Run YANG validate on generated config.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-config-engine/config_samples.py [hunk 1]
```diff
@@ -122,10 +132,6 @@ def generate_t1_smartswitch_switch_sample_config(data, ss_config):
 
         data['DHCP_SERVER_IPV4'] = {
             bridge_name: {
-                'customized_options': [
```
Comment (wen587) [inline src/sonic-config-engine/config_samples.py:125]: This has YANG depdency. Need to add together with depdency table

File: src/sonic-config-engine/config_samples.py [hunk 2]
```diff
@@ -50,7 +50,8 @@ def generate_t1_sample_config(data):
     data['DEVICE_METADATA']['localhost']['hostname'] = 'sonic'
     data['DEVICE_METADATA']['localhost']['type'] = 'LeafRouter'
     data['DEVICE_METADATA']['localhost']['bgp_asn'] = '65100'
-    data['LOOPBACK_INTERFACE'] = {"Loopback0|10.1.0.1/32": {}}
+    data['LOOPBACK_INTERFACE'] = {"Loopback0": {},
```
Comment (qiluo-msft) [inline src/sonic-config-engine/config_samples.py:53]: >LOOPBACK_INTERFACE [](http://example.com/codeflow?start=10&length=18)

This line is not related to SmartSwitch. Is it a generic bugfix? If yes, suggest use a standalone PR.

File: src/sonic-config-engine/tests/sample_output/t1-smartswitch.json [hunk 3]
```diff
@@ -379,6 +375,38 @@
 		}
 	},
 	"INTERFACE": {
+		"Ethernet0":{},
```
Comment (qiluo-msft) [inline src/sonic-config-engine/tests/sample_output/t1-smartswitch.json:378]: >Ethernet0 [](http://example.com/codeflow?start=3&length=9)

Does this PR enforce YANG validation? I am worried about future regression.

File: src/sonic-config-engine/config_samples.py (hunk 2 above)
Comment (wen587) [reply src/sonic-config-engine/config_samples.py]: Will use seperate PR

File: src/sonic-config-engine/tests/sample_output/t1-smartswitch.json (hunk 3 above)
Comment (wen587) [reply src/sonic-config-engine/tests/sample_output/t1-smartswitch.json]: Will use seperate PR

**Comment by wen587** [comment]
Hi @oleksandrivantsiv  could you help review

File: src/sonic-config-engine/config_samples.py [hunk 4]
```diff
@@ -85,6 +85,12 @@ def generate_t1_smartswitch_switch_sample_config(data, ss_config):
 
     bridge_name = 'bridge-midplane'
 
+    data['MID_PLANE_BRIDGE'] = {
+        "GLOBAL": {
+            "bridge": bridge_name,
+            "ip_prefix": "169.254.200.254/24"
+            }
```
Comment (qiluo-msft) [inline src/sonic-config-engine/config_samples.py:92]: wrong identation

**Review by qiluo-msft — APPROVED** [review]

**Review by ganglyu — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]
